label_sync/labels.md

```diff
@@ -0,0 +1,124 @@
+# Kubernetes GitHub Labels
+
+## Intro
+
+These file was auto generated by the [label_sync](https://git.k8s.io/test-infra/label_sync/) tool,
+based on the [labels.yaml](https://git.k8s.io/test-infra/label_sync/labels.yaml) that it uses to
+sync github labels across repos in the [kubernetes github org](https://github.com/kubernetes)
+
+### Why these labels?
+
+The rule of thumb is that labels are here because they are intended to be produced or consumed by
+our automation (primarily prow) across all repos. There are some labels that can only be manually
+applied/removed, and where possible we would rather remove them or add automation to allow a 
+larger set of contributors to apply/remove them.
+
+### How do I add a new label?
+
+- Add automation that consumes/produces the label
+- Open a PR against [labels.yaml](https://git.k8s.io/test-infra/label_sync/labels.yaml)
+- Involve [sig-contributor-experience](https://git.k8s.io/community/sig-contributor-experience) in the change, eg: chat about it in slack, mention @kubernetes/sig-contributor-experience-pr-reviews in the PR, etc.
+- After the PR is merged, a kubernetes CronJob is responsible for syncing labels daily
+
+
+## Labels that apply to both
```

Comment:
"both" isn't clear, because you haven't introduced the options yet. perhaps "Labels that apply to both issues and PRs"

Reply:
good point, I'll find a different way to generate these headings